### DISCUSSION TITLE
Feature request: Edit settings on the main screen

### DISCUSSION THREAD

**Discussion by IntendedPun** [Tyrrrz/LightBulb#277] [General]
I find myself fiddling with the settings throughout the day, depending on what I'm doing. It's just awkward to go clicking through different windows everytime I want to change something. It'd be useful to have an option to display some of the settings on the main screen, like sunrise/sunset and brightness/temperature. I made a couple of quick mockups of what I had in mind:
![LightBulb_design_01](https://github.com/Tyrrrz/LightBulb/assets/80411587/d2eb5233-d5d2-4604-adf4-216eca859903)
For the sunrise/sunset time, it could be either the fancy arrows that follow around the circle, or just the sliders around the text. I imagined the arrows making it possible to control the transition length as well.

Here's simpler one that'd probably be far easier to implement. Buttons on the side to show/hide the docks.
![LightBulb_design_02](https://github.com/Tyrrrz/LightBulb/assets/80411587/aef39ebf-5bd6-41c2-b1ff-3897f1d7ca06)

**Comment by Tyrrrz** [comment]
Sounds like a pretty interesting idea, although I don't think I will have the resources to implement it anytime soon.

**Reply by IntendedPun** [reply]
No prob! Thanks for making the software regardless! I've had it running for a couple of years now.
